crates/runtime/src/dataconnector/graphql.rs

```diff
@@ -771,4 +875,99 @@ mod tests {
             }
         }
     }
+
+    #[test]
+    fn test_json_object_unnesting() {
+        let unnest_parameters = super::UnnestParameters { depth: 100 };
+        let object = serde_json::from_str(r#"{"a": {"b": 1}}"#).expect("Invalid json");
```

Comment:
```suggestion
        let object = serde_json::from_str(r#"{"a": {"b": 1}}"#).expect("valid json");
```

nit: I read the string in the `expect` clause to be what I expect to happen.